### PR TITLE
Add base Zaius platform support

### PR DESCRIPTION
**Pull request by rlippert**
Adds the configs and XML package necessary to build a firmware image targeting the Zaius P9 server.

---
This change is [Reviewable](https://reviewable.io/reviews/open-power/op-build/780)

### CODE REVIEW

**Comment by e-liner** [comment]
ok to test

**Comment by pragupta** [comment]
ERROR: getAttribute(/sys-0/node-0/motherboard-0/proc_socket-0/module-0/p9_proc_l,XSCOM_BASE_ADDRESS) | Attribute not defined

Looks like we don't have a good xml.

**Comment by rlippert** [comment]
This version should be OK now.

(still tracking down problems with the latest serverwiz version of XML)

File: openpower/package/zaius-xml/zaius.mk [hunk 1]
```diff
@@ -0,0 +1,89 @@
+################################################################################
+#
+# zaius_xml
+#
+################################################################################
+
+ZAIUS_XML_VERSION ?= db53313f3873fc0a822121c0d95f257604387a76
+ZAIUS_XML_SITE ?= $(call github,rlippert,zaius-xml,$(ZAIUS_XML_VERSION))
```
Comment (williamspatrick) [inline openpower/package/zaius-xml/zaius.mk]: You want a repository created on open-power for zaius-xml?

**Comment by williamspatrick** [comment]
@dcrowell77 - Need someone from your team to review this also.

File: openpower/package/zaius-xml/zaius.mk (hunk 1 above)
Comment (stewart-ibm) [inline openpower/package/zaius-xml/zaius.mk]: We should probably have zaius-xml under open-power rather than @rlippert tree

Comment (rlippert) [inline openpower/package/zaius-xml/zaius.mk]: Yea that would be great as it was always the intent.  I think you can just clone my repository under the open-power github organization.

Comment (williamspatrick) [inline openpower/package/zaius-xml/zaius.mk]: Done.

**Comment by rlippert** [comment]
Updated pull request now to point to the official repository.

File: openpower/configs/hostboot/zaius.config [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+unset SFC_IS_IBM_DPSS
+set   ALLOW_MICRON_PNOR
+set   ALLOW_MACRONIX_PNOR
+
+# VPD options.
+set MVPD_READ_FROM_HW
+set MVPD_WRITE_TO_HW
+set MVPD_READ_FROM_PNOR
+set MVPD_WRITE_TO_PNOR
+unset DJVPD_READ_FROM_HW
```
Comment (dcrowell77) [inline openpower/configs/hostboot/zaius.config]: Is there a reason this isn't set?  Just bringup issues?

File: openpower/configs/hostboot/zaius.config [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+unset DJVPD_WRITE_TO_HW
+set DJVPD_READ_FROM_PNOR
+set DJVPD_WRITE_TO_PNOR
+unset CVPD_READ_FROM_HW
```
Comment (dcrowell77) [inline openpower/configs/hostboot/zaius.config]: CVPD_xxx has been replaced with MEMVPD_xxx

**Comment by stewart-ibm** [comment]
I'd like to get this in for the op-build v1.14 release due on 18 Jan.

@rlippert  / @dcrowell77  - you two think that's possible?

**Comment by stewart-ibm** [comment]
@rlippert the conflict in openpower/package/Config.in should be fairly easy to fix - it's related to the buildroot update we did recently, which changed slightly the way external trees are dealt with.

File: openpower/configs/hostboot/zaius.config (hunk 2 above)
Comment (rlippert) [inline openpower/configs/hostboot/zaius.config]: Yes it doesn't work yet as the necessary attributes are not present in zaius-xml yet.

File: openpower/configs/hostboot/zaius.config (hunk 3 above)
Comment (rlippert) [inline openpower/configs/hostboot/zaius.config]: Fixed to use new MEMVPD config symbol.

**Comment by rlippert** [comment]
Rebased and rebuilt, should be OK to merge now.

**Comment by e-liner** [comment]
I'm going to retest this once with zaius, just to be sure.

**Comment by e-liner** [comment]
retest this please

**Comment by e-liner** [comment]
retest this please

**Comment by stewart-ibm** [comment]
retest this please

**Comment by stewart-ibm** [comment]
(we seem to be having some issues with one of the IPL tests... if it doesn't magically fix itself, hopefully @e-liner can poke it with an appropriately sharp stick or try other forms of percussive maintenance).

**Comment by e-liner** [comment]
retest this please

**Comment by e-liner** [comment]
retest this please

**Comment by e-liner** [comment]
If this fails again, I'm removing one of our hardware tests because it looks like there's something wrong with the machine.l

**Comment by stewart-ibm** [comment]
This time github failed :(

**Comment by stewart-ibm** [comment]
retest this please

**Comment by e-liner** [comment]
I got this working with a rebuild, now we just need github to get the response.

**Comment by e-liner** [comment]
retest this please

**Comment by e-liner** [comment]
retest this please.

**Pull request merged**